components/raftstore/src/store/mod.rs

```diff
@@ -21,7 +21,7 @@ mod peer_storage;
 mod read_queue;
 mod region_snapshot;
 mod replication_mode;
-mod snap;
+pub mod snap;
```

Comment:
Why?

Reply:
We need some functions inside to manipulate snapshots, including `snap::snap_io::get_decrypter_reader` and `snap::plain_file_used`